### PR TITLE
(For Mohammed) Add get session decisions

### PR DESCRIPTION
**Pull request by mlegore**
*Purpose:* 
- Adds support for get session level decisions.

*Technical overview:*
- Updates client methods to call new endpoint (https://siftscience.com/developers/docs/java/decisions-api/decision-status),

**Testing Plan**
- Expanded and added unit tests and run them locally.

**Deployment**
- Publish gems
- Github release

### CODE REVIEW

**Comment by mjouahri** [comment]
Can you fill up the description section please. Taking a look at the code now.

File: HISTORY [hunk 1]
```diff
@@ -1,3 +1,6 @@
+=== 3.1.0 2018-06-04
+- Adds support for sessions decisions to [Decisions API](https://siftscience.com/developers/docs/curl/decisions-api)
```
Comment (mjouahri) [inline HISTORY]: Maybe "get session decisions" is better wording. Depends are we returning all the decisions associated with a session or just the latest ?
Also do we have support for batch get sessions or just get session ?

File: lib/sift/client.rb [hunk 2]
```diff
@@ -504,6 +504,45 @@ def get_order_decisions(order_id, opts = {})
...
+    #   The ID of the user in the session.
+    #
+    # session_id::
+    #   The ID of a session.
+    #
+    # opts (optional)::
+    #   A Hash of optional parameters for this request --
+    #
+    #   :account_id::
+    #     Overrides the API key for this call.
```
Comment (mjouahri) [inline lib/sift/client.rb]: Can you update this to account id

File: HISTORY (hunk 1 above)
Comment (mlegore) [reply HISTORY]: It's just get session, not batched

**Pull request merged**